### ISSUE TITLE
개발관련 - 나코딩한다 개발규칙 정립과 가이드 제작하기

### ISSUE THREAD

**Issue by binsoopark** [gdgsuwon/solocoding2019#7]
1. 행사 시작 전에 개발을 진행하는 방법에 대한 정립
2. 정립한 방법에 대해 실제 진행시 문제가 없을지에 대해 테스트를 통한 검증 필요
3. git을 commit하고 push하는 공통의 컨벤션을 정리하여 초보자도 쉽게 따라할 수 있도록 가이드 만들기(스크린샷 포함)

**Comment by binsoopark** [comment]
- github에 중앙 repository를 만들고 각 신청자별 닉네임(혹은 이름)을 기준으로 branch를 만들어준다.
- 각 진행자별 commit/push를 이용하여 자신의 상태를 중앙 repository에 저장할 수 있도록 한다.
- 완료되면, 각 branch를 master로 merge하고, 이후 공유 행사에서 공유될 수 있도록 한다.

**Comment by yjaeseok** [comment]
1. git client 설치
2. git config 설정
3. github 가입
4. git clone
5. git checkout
6. gitignore 설정
7. git add
8. git commit
9. git push
10. flutter 다운로드
11. flutter doctor
12. flutter ide
13. new project
14. hello world
15. commit
16. push

**Comment by yjaeseok** [comment]
이정도 순서로 가이드문서를 작성하면될것 같은데~~ 혹시 더 넣을게있을까용?!

**Comment by binsoopark** [comment]
14번 hello world가 program run을 의미하는 것이라면... 뭐 거의 완벽에 가까운 수준이군요.ㅎㅎㅎ
이 가이드 완료 이후 개발 진행 + 15,16번 반복으로 처리될 수 있으니, 필요한 지식에 대한 가이드는 충분할 것 같습니다.
순서 괜찮은 것 같네요~

**Comment by yjaeseok** [comment]
네 이순서로 좀 다시 정리를 해볼게요 ㅋ 어제 해놓은거보니 순서가 뒤죽박죽이라 일단 순서부터 고민해보고있었어요 ㅋ
